src/base-components/switch-base/switch-base.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './switch-base.css'
+
+const _switchClasses = () => ({
+  checkbox: 'w-0 h-0 hidden',
+  disabled: 'opacity-50 cursor-not-allowed',
```

Comment:
In the design file, Do had a specific design for disabled states. I didn't see a screenshot on Chromatic so I'm not sure if you already followed that. Otherwise radio buttons should change colour to `#BDBDBD` (Secondary/400 in the design system) when disabled.
![Screen Shot 2021-06-23 at 4 01 11 PM](https://user-images.githubusercontent.com/33563070/123160507-47a42c80-d43c-11eb-9131-983948a554b0.png)